include/ipmitool/ipmi.h

```diff
@@ -283,7 +283,7 @@ typedef enum IPMI_OEM {
      IPMI_OEM_MAGNUM     = 5593,
      IPMI_OEM_TYAN       = 6653,
      IPMI_OEM_QUANTA     = 7244,
-     IPMI_OEM_NEWISYS    = 9237,
+     IPMI_OEM_VIKING_ENTERPRISE_SOL = 9237,
```

Comment:
Let's please limit this to just `IPMI_OEM_VIKING`. I don't see any other vikings around, so let's be terse.